Revert "Introduce APICAST_LUA_SOCKET_KEEPALIVE_REQUESTS environment variable" (#1504)
3scale/APIcast: doc/build.md

## Release

APIcast is released as [Docker image](https://docs.docker.com/engine/tutorials/dockerimages/). 

## Dependencies

> APIcast uses LuaRocks, the package manager for Lua modules, to install dependencies. With a correct configuration, LuaRocks installs dependencies into the correct path where OpenResty can see them. 

Reviewer: ```suggestion
APIcast uses LuaRocks, the package manager for Lua modules, to install dependencies. With the proper configuration, LuaRocks installs dependencies into the path accessible to OpenResty. 
```
